docs/configuring.rst

```diff
@@ -174,6 +175,14 @@ Miscellaneous options
     :type: string
     :default: ``#2980B9``
 
+.. confval:: flyout_display
+
+    Specify how to display the flyout (language and version selector).
+    This can be either ``attached`` or ``hidden``.
```

Comment:
Would be useful to define what `attached` means.

```suggestion
    This can be either ``attached`` or ``hidden``.
    ``attached`` means that it will show the flyout in the bottom of the sidebar.
    You will need to disable the default `Read the Docs flyout <https://docs.readthedocs.io/en/stable/flyout-menu.html>`_ in order to not have 2 flyouts showing.
```